.changeset/sharp-items-talk.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/shopify-app-react-router': patch
```

Comment:
We don't need a changeset, as nothing in the actual package changed. 

You can either add an empty changeset. ```pnpm add changeset --empty` or add the skip changelog label to this PR.